user_docs/en/changes.md

```diff
@@ -6,6 +6,8 @@
 
 ### New Features
 
+* New key commands:
+  * Added unassigned commands for vertical and horizontal rotation of the mouse wheel to enhance navigation on web pages and apps with dynamic content, such as Dism++. (#16462, @Cary-Rowen)

```

Comment:
Not totally convinced with this one. I do not know how horizontal scrolling is actually performed with a physical mouse. Anybody has an idea?

Reply:
Before implementing this PR, I asked visual users how horizontal scrolling with a mouse wheel is typically implemented:
1. Some software supports horizontal scrolling by pressing SHIFT + Mouse Wheel, such as WPS, Visio, and Chromium-based browsers.
2. There is also some software that uses Ctrl + Shift + Mouse Wheel for horizontal scrolling, such as Microsoft Excel.
3. Limitation: Chromium-based browsers support Shift + Mouse Wheel scrolling on pages with horizontal scroll bars, but this does not work on pages without horizontal scroll bars.
4. For some laptop touchpad drivers, flicking left and right with two fingers can achieve the same behavior as horizontal scrolling with a mouse wheel.
5. When you type `main.cpl` in the Windows Run dialog and press Enter, you can open the mouse settings. There are horizontal scrolling options under the "Scroll Wheel" tab.

However, Microsoft's physical mouse can tilt the mouse wheel for horizontal scrolling:
https://support.microsoft.com/en-us/topic/what-are-the-default-buttons-and-wheel-assignments-for-my-mouse-47f656a8-72d0-63ec-8d1d-dda7122bef6b

Reply:
I think this one is missing:
Most mouses have a scroll wheel click, by pressing down on the scroll wheel. Holding this allows you to scroll horizontally and vertically at the same time by moving the cursor in the direction you want to scroll